.travis.yml

```diff
@@ -0,0 +1,32 @@
+branches:
+  only:
+  - trunk
```

Comment:
You can leave out the limitation to only `trunk`, because this build should also run on feature branches and on the `master` branch, which this repo will probably get as soon as it's no longer tracking the SVN repo.

No need to worry about the `cron` branch: Travis CI will only build based on the `.travis.yml` in the current branch (more specifically the current commit). And this `.travis.yml` won't end up in the `cron` branch.